Cargo.toml

```diff
@@ -18,3 +18,16 @@ codegen-units = 1
 lto = "fat"
 debug = false
 inherits = "release"
+
+[workspace.lints.rust]
+unsafe_code = "deny"
+# lower the priority of rust_2024_compatibility to allow overriding the lints it includes
+rust_2024_compatibility = { level = "warn", priority = -1 }
```

Comment:
I think we are already on rust 2024 so this warning can be a warning and cause the builds to fail